examples/multiple-params/.eslintrc.js

```diff
@@ -1,3 +1,12 @@
 module.exports = {
   extends: ["@remix-run/eslint-config"],
+  overrides: [
+    {
+      // all ```jsx & ```tsx code blocks in .md files
+      files: ["**/*.md/*.jsx", "**/*.md/*.tsx"],
+      rules: {
+        "react/jsx-no-undef": "off",
+      },
+    },
+  ],
```

Comment:
If there's another way to not make ESLint error on `examples/multiple-params/README.md`, I'd like to do it that way, so all examples just extend `@remix-run/eslint-config`.
https://github.com/remix-run/remix/runs/5416525002?check_suite_focus=true#step:6:1486

I've been looking into making the root rule more important, but couldn't find any better way unfortunately 😕